### ISSUE TITLE
TensorFlow 1.13.1 released

### ISSUE THREAD

**Issue by l1t1** [leela-zero/leela-zero#2255]
Major Features and Improvements
    TensorFlow Lite has moved from contrib to core. This means that Python modules are under tf.lite and source code is now under tensorflow/lite rather than tensorflow/contrib/lite.
    TensorFlow GPU binaries are now built against CUDA 10 and TensorRT 5.0. -->maybe helpful to leelaz train
    Support for Python3.7 on all operating systems.
    Moved NCCL to core.

**Comment by wonderingabout** [comment]
thanks @l1t1

**Comment by wonderingabout** [comment]
@l1t1 

cudnn 7.5 released : 

https://developer.nvidia.com/cudnn

Up to 3x faster training of ResNet-50 and GNMT on Tesla V100 vs. Tesla P100
Improved depth-wise separable convolution for training models such as Xception and Mobilenet
Multi-Head Attention for accelerating popular models such as Transformer
New tensor folding APIs for accelerated performance on models such as Mask R-CNN, GANs and DeepSpeech2
Read the latest cuDNN release notes for a detailed list of new features and enhancements. https://docs.nvidia.com/deeplearning/sdk/cudnn-release-notes/

**Comment by l1t1** [comment]
https://github.com/Tencent/PhoenixGo/pull/92#issuecomment-468920340 seems good

**Comment by wonderingabout** [comment]
@l1t1 

better look at this : 

on gtx1060 75w
```
batch size 4 no tensorrt :       131 n/s vs 157 n/s
batch size 4 tensorrt 3.0.4 :    153 n/s vs ???
batch size 16 no tensorrt :      216 n/s vs 270 n/s
```

the gains for tesla v100 and other higher cards are probably much bigger, as was tested in tensorflowf 1.8.0

batch size 4 to 16 (no tensorrt)
```
gtx 1060 75w : + 25%
tesla v100 :      +135%
```

**Comment by wonderingabout** [comment]
also you can close this thread i think, it doesnt look very relevant for leela zero :+1:

**Comment by l1t1** [comment]
ok

**Comment by l1t1** [comment]
TensorFlow Lite Android nightly runs inference x2 faster than latest version 1.13.1

https://github.com/tensorflow/tensorflow/issues/26385
